tests/airgapped/README.md

```diff
@@ -27,35 +23,30 @@ You can run the script that will spin up an airgapped microk8s cluster with:
 ```bash
 ./tests/airgapped/airgap.sh \
   --node-name airgapped-microk8s \
-  --microk8s-channel 1.25-strict/stable \
-  --bundle-path releases/1.8/stable/kubeflow/bundle.yaml \
-  --juju-channel 3.1/stable
+  --microk8s-channel 1.29-strict/stable \
+  --bundle-path releases/1.9/stable/bundle.yaml \
+  --juju-channel 3.4/stable
 ```
 
 ### Size considerations
 
-As stated in the beginning these scripts require a lot of storage, if run with
-the full set of images of Kubeflow. To better expose this, we'll take for
-granted that the total of all OCI images of Kubeflow is 125Gb. Then the amount
-of storage needed is:
-- 125Gb, for host to pull all images locally
-- 125Gb, for the compressed `images.tar.gz` (the size almost always will be
-  smaller, but here I'll use the worst case scenario
-- 125Gb, to copy this tarbal inside the airgapped LXC machine
-- 125Gb, to copy the contents of the tarball into the container registry inside
-  the airgapped LXC machine
-
-So in the worst case, we need to have at least 500Gb to be able to run those
+The total size of all OCI images needed for Kubeflow is around 125 GB. As a result, the total amount of storage needed for the airgapped test is:
+- 125 GB, for the host to pull all images locally
+- 125 GB, for the compressed `images.tar.gz` (the size almost always will be smaller, so this is the worst case scenario)
+- 125 GB, to copy `images.tar.gz` inside the airgapped LXC machine
+- 125 GB, to copy the contents of the tarball into the container registry inside the airgapped LXC machine
```

Comment:
I think it will be less than 125Gb now that seldon is not part of the bundle, can you check?

Reply:
yes, let's change it to that